Containerfile.rag

```diff
@@ -1,21 +1,18 @@
 # vim: set filetype=dockerfile
-
-# port, commands, and other details inherited from the base image
 ARG OLS_VERSION=latest
+ARG OCP_VERSION=4.15
+ARG LIGHTSPEED_RAG_CONTENT_DIGEST=sha256:eb3472dd675cc79b02e4d15b53b12c9c365d6c2335a6026f55de443dd95c902f
+
+FROM quay.io/openshift/lightspeed-rag-content@${LIGHTSPEED_RAG_CONTENT_DIGEST} as lightspeed-rag-content
```

Comment:
Just wondering, if this image already has embeddings stored in vector db or just the plain docs? I hope it only has embeddings and that too are generated as a Day 1 operation (.i.e just like a one time activity)

Reply:
The lightspeed-rag-content image has the embedding model and the vector db it. The plaintext docs are in the openshift/lightspeed-rag-content repo.

Reply:
that image has the generated embeddings.

the idea is that we will periodically generate a new embedding image and then update the SHA in this file to pick up the new embeddings.

Reply:
Understood. Thanks for confirming.